### PR TITLE
feat: add frame drop issue generation

### LINKED ISSUE #64: Simulate a "Frame Drop" profiling issue

### PR DESCRIPTION
**Pull request by armcknight**
Closes #64 

adds a couple other quality of life improvements:

- remove cocoapods and bring in Sentry with SPM (removes lots of files, including the Sentry.xcworkspace). you can right click on the packages and select "Update..." to upgrade them: 
image

launch args: 
- debug logging overrides via launch args, in both directions, depending on whether it's a debug build (default to debug mode on, with an override option to disable) or release (vice versa)
- add launch arg option to disable SDK swizzling, and set a tag on the scope indicating whether it's enabled or not (helps if trying to figure out interactions with various parts of the automatic instrumentation, or just turn them off for something)
image

### CODE REVIEW

**Comment by armcknight** [comment]
I was just waiting to see if i could investigate a possible bug in the sdk related to frame drop issues but that shouldn't hold this up any more, I'd say this is ready to go.

**Review by realkosty — APPROVED** [review]

**Comment by armcknight** [comment]
I reverted the change to CODEOWNERS and put it in a separate PR: https://github.com/sentry-demos/ios/pull/66/files

**Comment by kpujjigit** [comment]
Flagship Error Symbolicated:
image

Flagship Transaction as expected:
image

Changes Verified.

**Pull request merged**
